src/components/BuyListElement/BuyListElement.module.css

```diff
@@ -0,0 +1,13 @@
+.imageBuyListElement{
```

Comment:
It is better to name css classes using "kebab-case" (hyphen method)
For example: **image-buy-list-element**

 Anyways, you can keep the current naming, but keep in mind for the future